src/main/cpp/ShooterBehaviour.cpp

```diff
@@ -6,33 +6,26 @@ ShooterManualControl::ShooterManualControl(Shooter *shooter, frc::XboxController
 }
 
 void ShooterManualControl::OnTick(units::second_t dt) {
-    if (_codriver.GetLeftBumper()) {
-       _shooter->setState(ShooterState::kSpinUp);
-       _shooter->setPidGoal(65_rad_per_s);
-    } else if (_codriver.GetRightBumper()) {
-       _shooter->setState(ShooterState::kSpinUp);
-       _shooter->setPidGoal(20_rad_per_s);
-    }
-    
-    double rawControl;
+
+
     if(_codriver.GetAButtonPressed()){
-        if (rawControl == true){
-            rawControl = false;
+        if (_rawControl == true){
+            _rawControl = false;
         } else {
-            rawControl = true;
+            _rawControl = true;
         }
     }
-if (!rawControl) { // rawControl == false
-    if (_codriver.GetLeftBumper()) {
-        _shooter->setState(ShooterState::kSpinUp);
-        _shooter->setPidGoal(65_rad_per_s);
-    } else if (_codriver.GetRightBumper()) {
-        _shooter->setState(ShooterState::kSpinUp);
-        _shooter->setPidGoal(20_rad_per_s);
+    if (!_rawControl) { // rawControl == false
+        if (_codriver.GetLeftBumper()) {
+            _shooter->setState(ShooterState::kSpinUp);
+            _shooter->setPidGoal(65_rad_per_s);
+        } else if (_codriver.GetRightBumper()) {
+            _shooter->setState(ShooterState::kSpinUp);
+            _shooter->setPidGoal(20_rad_per_s);
+        }
+    } else {
+         _shooter->setRaw(_codriver.GetRightBumperPressed() * 12_V);
```

Comment:
bumper is a button which means it either returns a true or a false, you should use something that returns an analog value (something that has 0-1 values) such as a trigger or a joystick